packages/amplify-frontend-javascript/index.js

```diff
@@ -10,6 +11,8 @@ const { createAWSExports, deleteAmplifyConfig } = require('./lib/frontend-config
 
 const pluginName = 'javascript';
 
+const emptyAwsExportsPath = path.join(__dirname, 'lib', 'aws-exports.empty.js');
```

Comment:
This location is consistent with how the rest of the package is structured. Not arguing that it shouldn't be moved to resources in the future, but I think this should stay for now as it would require larger changes to the package (this package is still js)